sql/mob_groups.sql

```diff
@@ -11630,7 +11631,7 @@ INSERT INTO `mob_groups` VALUES (13756,4702,91,0,128,0,0,0,72,73,0);
 INSERT INTO `mob_groups` VALUES (13757,4701,91,0,128,0,0,0,95,96,0);
 INSERT INTO `mob_groups` VALUES (13758,4700,171,0,128,0,0,0,97,98,0);
 INSERT INTO `mob_groups` VALUES (13759,4699,171,0,128,0,0,0,95,96,0);
-INSERT INTO `mob_groups` VALUES (13760,4698,171,0,128,4583,0,5000,82,83,0); -- Mortille Mortelle
```

Comment:
What likely happened is you replaced a file from a newer revision (your fork) with your edited one that was from an older revision (your local copy). 

Git+GitHub don't like that.  Just restore the line to what it was and when I collapse changes it'll be fine.